platform/android/export/export.cpp

```diff
@@ -2491,12 +2547,21 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 			//stores all the project files inside the Gradle project directory. Also includes all ABIs
 			_clear_assets_directory();
+			_remove_copied_libs();
 			if (!apk_expansion) {
 				print_verbose("Exporting project files..");
-				err = export_project_files(p_preset, rename_and_store_file_in_gradle_project, nullptr, ignore_so_file);
+				CustomExportData user_data;
+				user_data.debug = p_debug;
+				err = export_project_files(p_preset, rename_and_store_file_in_gradle_project, &user_data, copy_gradle_so);
 				if (err != OK) {
 					EditorNode::add_io_error("Could not export project files to gradle project\n");
 					return err;
+					if (user_data.libs.size() > 0) {
+						FileAccessRef fa = FileAccess::open("res://android/build/libs/gdnativelibs.json", FileAccess::WRITE);
```

Comment:
Can you turn `"res://android/build/libs/gdnativelibs.json"` into a constant since it's used in a couple of locations.

Reply:
Done.